src/js/components/covid19/DataSourcesAndMethodologiesPage.jsx

```diff
@@ -122,6 +135,14 @@ export default () => {
                     <div className="about-content">
                         <div className="about-padded-content">
                             <div className="about-section-wrapper" id="data-sources-datasets">
+                                <div className="back-link">
+                                    <a
+                                        href={getBaseUrlNoHash(slug)}
```

Comment:
can we change this to `href="/#/disaster/covid-19"` so we do not refresh the page.